common/app/agents/DeeplyReadAgent.scala

```diff
@@ -33,40 +33,54 @@ class DeeplyReadAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi) ex
      */
     Future
       .sequence(Edition.allEditions.map { edition =>
-        ophanApi.getDeeplyRead(edition).flatMap {
-          ophanDeeplyReadItems =>
-            log.info(s"ophanItems updated with: ${ophanDeeplyReadItems.size} new items")
-            val constructedTrail: Seq[Future[Trail]] = ophanDeeplyReadItems.map {
-              ophanItem =>
-                log.info(s"CAPI lookup for Ophan deeply read item: ${ophanItem.toString}")
-                val path = removeStartingSlash(ophanItem.path)
-                log.info(s"CAPI Lookup for path: $path")
-                val capiRequest = contentApiClient
-                  .item(path)
-                  .showTags("all")
-                  .showFields("all")
-                  .showReferences("none")
-                  .showAtoms("none")
-                val trailFromCapiResponse = contentApiClient
-                  .getResponse(capiRequest)
-                  .map { res =>
-                    res.content.flatMap { capiData =>
-                      log.info(s"Retrieved CAPI data for Deeply Read item: ${path}")
-                      deeplyReadUrlToTrail(capiData)
+        ophanApi
+          .getDeeplyRead(edition)
+          .flatMap {
+            ophanDeeplyReadItems =>
+              log.info(s"Fetched ${ophanDeeplyReadItems.size} Deeply Read items for ${edition.displayName}")
+              val constructedTrail: Seq[Future[Option[Trail]]] = ophanDeeplyReadItems.map {
+                ophanItem =>
+                  log.info(s"CAPI lookup for Ophan deeply read item: ${ophanItem.toString}")
+                  val path = removeStartingSlash(ophanItem.path)
+                  log.info(s"CAPI Lookup for path: $path")
+                  val capiRequest = contentApiClient
+                    .item(path)
+                    .showTags("all")
+                    .showFields("all")
+                    .showReferences("none")
+                    .showAtoms("none")
+
+                  contentApiClient
+                    .getResponse(capiRequest)
+                    .map { res =>
+                      res.content.flatMap { capiData =>
+                        log.info(s"Retrieved CAPI data for Deeply Read item: ${path}")
+                        deeplyReadUrlToTrail(capiData)
+                      }
+                    }
+                    .recover {
+                      case NonFatal(e) =>
+                        log.error(s"Error retrieving CAPI data for Deeply Read item: ${path}. ${e.getMessage}")
+                        None
                     }
-                  }
-                  .recover {
-                    case NonFatal(e) =>
-                      log.error(s"Error retrieving CAPI data for Deeply Read item: ${path}. ${e.getMessage}")
-                      None
-                  }
-                trailFromCapiResponse.map(_.get)
```

Comment:
Thanks to @jonathonherbert and @alinaboghiu for confirming my hunch that this was a real issue!